.readthedocs.yml

```diff
@@ -33,6 +33,7 @@ build:
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
   configuration: docs/conf.py
+  fail_on_warning: false
```

Comment:
I know warnings can be annoying. Would it be viable to fix them properly?

Reply:
It looks like currently there is a hard error?

-- https://readthedocs.org/projects/wetterdienst/builds/24710976/

Reply:
Yep, the `fail_on_warning` could be reverted, after the warning source was fixed.

Reply:
The second: I wanted to bump poetry for docs, but it seems the new poetry version won't install all necessary dependencies...

Reply:
Ah found the issue here: https://github.com/python-poetry/poetry/issues/9025